scripts/globals/spells/dia_iii.lua

```diff
@@ -44,12 +42,8 @@ function onSpellCast(caster, target, spell)
     local final = finalMagicAdjustments(caster, target, spell, dmg)
 
     -- Calculate duration and bonus
-    local duration = calculateDuration(caster:getMerit(tpz.merit.DIA_III), spell:getSkillType(), spell:getSpellGroup(), caster, target)
+    local duration = calculateDuration(180, spell:getSkillType(), spell:getSpellGroup(), caster, target)
     local dotBonus = caster:getMod(tpz.mod.DIA_DOT) -- Dia Wand
-    -- If caster has the spell but no merits in it, they are either a mob or we assume they are GM or otherwise gifted with max duration
-    if duration == 0 then
-        duration = 150
```

Comment:
[Dia III might be a straight 180 seconds now along with Bio III](https://forum.square-enix.com/ffxi/threads/55751-August.-6-2019-%28JST%29-Version-Update?p=618448&viewfull=1#post618448)
> The duration of the white magic spell Dia III has been adjusted to 180 seconds.

I don't know if that means it floors at 180 though, might go below. Pinging @project-topaz/researcher if they feel like casting Dia III on an ilvl mob while naked.

Reply:
For enfeebling magic, the calculateDuration() will

* apply saboteur bonus (+25% vs NMs, otherwise +100%)
* apply bonus from ENFEEBLING_MAGIC_DURATION merit.

Reply:
Oh I see; I was mentally linking Bio and Dia, and saw Bio didn't use the function (retrospect: because it's not enfeebling), so thought calculateDuration was for the purpose of taking resistances into play or something.